notebooks/composer/composer_vertex/vertex_upload_model_docker/build_image.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+gcloud builds submit --config=cloudbuild.json .
```

Comment:
Same comments: `cloudbuild.json` -> `cloudbuild.yaml`; put instruction in the notebook & rm file.